src/ansys/fluent/core/quantity.py

```diff
@@ -0,0 +1,554 @@
+from collections import OrderedDict
+
+
+class UnitsTable(object):
+    _fundamental_units = {
+        "kg": "M",
+        "g": "M",
+        "lb": "M",
+        "lbm": "M",
+        "slug": "M",
+        "m": "L",
+        "cm": "L",
+        "ft": "L",
+        "in": "L",
+        "s": "T",
+        "A": "A",
+        "K": "K",
+        "C": "K",
+        "F": "K",
+        "R": "K",
+        "mol": "mol",
+        "cd": "cd",
+        "sr": "sr",
+        "radian": "Angle",
+        "degree": "Angle",
+    }
+
+    _derived_units = {
+        "N": "kg m s^-2",
+        "Pa": "N m^-2",
+        "W": "N m s^-1",
+        "J": "N m",
+        "V": "A ohm",
+        "F": "N m V^-2",
+        "H": "N m A^-2",
+        "S": "ohm^-1",
+        "Wb": "N m A^-1",
+        "T": "Wb m^-2",
+        "dyne": "g cm s^-2",
+        "erg": "dyne cm",
+        "pdl": "lbm ft s^-2",
+        "BTU": "J",
+        "psi": "lbf in^-2",
+        "lbf": "slug ft s^-2",
+        "psf": "lbf ft^-2",
+        "ohm": "kg m^2 s^-3 A^-2",
+        "Hz": "s^-1",
+    }
+
+    _derived_units_with_conversion_factor = {
+        "l": (0.001, "m^3"),
+        "gal": (0.0037854117839999993, "m^3"),
+        "BTU": (1055.056, "J"),
+    }
+
+    _multipliers = {
+        "d": 10**-1,
+        "c": 10**-2,
+        "m": 10**-3,
+        "μ": 10**-6,
+        "n": 10**-9,
+        "p": 10**-12,
+        "f": 10**-15,
+        "a": 10**-18,
+        "z": 10**-21,
+        "y": 10**-24,
+        "da": 10**1,
+        "h": 10**2,
+        "k": 10**3,
+        "M": 10**6,
+        "G": 10**9,
+        "T": 10**12,
+        "P": 10**15,
+        "E": 10**18,
+        "Z": 10**21,
+        "Y": 10**24,
+    }
+
+    _dimension_order = OrderedDict(
```

Comment:
The second entry of each tuple seems to be a short name for each dimensional quantity.  I think you would want to avoid using actual units for that.  eg: `K`, `mol`, `cd`, `A` and `sr` should probably be something else no?

Reply:
For chemical amount I think you want capital `N` no? 

https://en.wikipedia.org/wiki/Dimensional_analysis

Reply:
Done. Thank you.